app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/MessageService.scala

```diff
@@ -84,6 +91,10 @@ class MessageService @Inject() (
       .map(_.getOrElse(UpdateOutcome.Locked))
   }
 
+  private def shouldMigrateLastUpdated(maybeLastRetrieved: Option[Instant]): Boolean =
+    //noinspection MapGetOrElseBoolean
+    maybeLastRetrieved.map(_.isBefore(migrateLastUpdatedCutoff)).getOrElse(true)
```

Comment:
Should this be `getOrElse(false)`? This would mean we run the migration if we've never polled for them before, at which point there should be nothing to migrate?

Reply:
You're right, this should be false, fixing